src/sentry/utils/assets.py

```diff
@@ -1,5 +1,21 @@
+from __future__ import annotations
+
+import os.path
+
+from cachetools.func import ttl_cache
 from django.conf import settings
 
+from sentry.utils import json
+
+
+@ttl_cache(ttl=60)
```

Comment:
The `ttl` is in seconds right? This means when a frontend deploy complete's we'll need to wait up to 1 minute for the changes to show up right